catalog/app/components/JsonEditor/Input.js

```diff
@@ -87,18 +116,13 @@ export default function Input({
   )
 
   const onBlur = React.useCallback(() => {
-    if (columnId !== COLUMN_IDS.KEY || isString(value)) {
+    if (columnId !== COLUMN_IDS.KEY || typeof value === 'string') {
```

Comment:
`typeof value === 'string'` is unsafe -- it fails if the value is, for example, `new String('')`. `R.is(String, value)` is safer. When I told that we should use inline checks instead of `lodash.isX`, I meant the equality checks (`=== null`, `=== undefined), not type checks, bc `typeof` frequently behaves counter-intuitively.

Reply:
> typeof value === 'string' is unsafe -- it fails if the value is, for example, new String('').

Good question for a job interview, right after "What's the result of `[] + {}` and `{} + []`" :)

---

Fixed, thanks